pkg/porter/explain_test.go

```diff
@@ -198,6 +198,13 @@ func TestExplain_generatePrintableBundleParams(t *testing.T) {
 				Required:   true,
 			},
 		},
+		Custom: map[string]interface{}{
```

Comment:
I don't think this change is used by the test since it's verifying bundle parameters. So this can be removed.

Reply:
The reason why I added the `Custom` property to the bundle is because it yields panic if it's not present when we want to load the stamp when calling `generatePrintable`. 

![image](https://user-images.githubusercontent.com/7043511/107848362-64dd4480-6e25-11eb-8572-dd97a67c2042.png)

Reply:
Oops, sorry about that you are right. 👍